TARS/src/common/ServoControl.cpp

```diff
@@ -26,12 +30,12 @@ ServoControl::ServoControl(PWMServo* servo) { servo_ = servo; }
  */
 void ServoControl::roundOffAngle(float& value) {
```

Comment:
IMO, this really shouldn't take a reference. It should take a `float` and return some sort of integer.. I will die on this hill.

Also: this function will have a fit if you pass in a `NaN`. `NaN` return false for all comparison operations, so it won't be replaced. `std::round` returns a `NaN` if it's argument is `NaN`, and a `NaN` coerced to an integer becomes `INT_MIN`. And a `NaN` being passed into the function is a distinct possibility since the argument to this function will come from the result of `asin`. We had this problem once when we were testing, I believe. It was  of course resolved by fixing the `asin` fit we used, but having some safety margin in this function as well could be useful.